tutorials/qdrant_langchain_instrumentation_search_and_retrieval_tutorial.ipynb

```diff
@@ -0,0 +1,478 @@
+{
```

Comment:
<p>This cell is no longer necessary since we compute ranking metrics automatically upon ingestion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>This is a good suggestion but I don't think I'm going to change the flow of the notebook here to use our entire <code>log_evaluations</code> flow, let's leave that demonstration for the more targeted examples </p>